Joel/HYFcalculator.html

```diff
@@ -0,0 +1,161 @@
+    <button class="btn btn-primary" onclick="divide()">Divide</button>
+    <button class="btn btn-primary" onclick="add()">Add</button>
+    <button class="btn btn-primary" onclick="subtract()">Subtract</button>
+    <button class="btn btn-primary" onclick="largest()">Largest</button>
+    <button class="btn btn-primary" onclick="closestTo100()">Closest to 100</button>
+
+    <h2>Result:</h2>
+
+    <h3 id="js-result">
+
+      <!-- WRITE THE RESULT HERE -->
+      <!-- hint: use innerHTML -->
+
+    </h3>
+
+  </div>
+
+</body>
+
+<script>
+
+  function multiply() {
+
+    // Here we look inside our HTML document and get the element with ID 'number1'
+    // which is an input field, so we can get its value (what we have typed inside the input field)
+    num1 = document.getElementById('number1').value;
+
+    // We do the same for the second input field
+    num2 = document.getElementById('number2').value;
+
+    // HINT: console.log the values here to see if the above is working
+
+    // we multiply the two values
+    result = num1 * num2;
+
+    // We select the element where we want to write the result
+    var resultElement = document.getElementById('js-result');
+    // we write the result inside the HTML of the element
+    resultElement.innerHTML = result;
+  }
+
+  function divide() {
+    
+    num1 = document.getElementById('number1').value;
+    num2 = document.getElementById('number2').value;
+    
+    result = num1 / num2;
+    
+    var resultElement = document.getElementById('js-result');
+    resultElement.innerHTML = result;
+  }
+
+  function add() {
+    
+    num1 = document.getElementById('number1').value;
+    num2 = document.getElementById('number2').value;
+    
+    result = (+num1) + (+num2); /* //My numbers where treated as strings. The solution I found online was to use the unary plus operator, 
+    but although it works, Im not sure is the best solution */
+    
+    var resultElement = document.getElementById('js-result');
+    resultElement.innerHTML = result;
+    console.log(result);
+  }
+
+  function subtract() {
+     
+    num1 = document.getElementById('number1').value;
+    num2 = document.getElementById('number2').value;
+    
+    result = num1 - num2;
+    
+    var resultElement = document.getElementById('js-result');
+    resultElement.innerHTML = result;
+  }
+
+  function largest() {
+     
+    num1 = document.getElementById('number1').value;
+    num2 = document.getElementById('number2').value;
+    
+    if (num1 > num2) {
+      result = num1;
+    } else if (num1 < num2){ 
+      result = num2;
+    } else {
+      result ="Both numbers are equal"
+    }
+    var resultElement = document.getElementById('js-result');
+    resultElement.innerHTML = result;
+  }
+
+  function closestTo100() {
+
+    num1 = document.getElementById('number1').value;
+    num2 = document.getElementById('number2').value;
+    let num1Input = Math.abs(Math.abs(num1) - 100); 
```

Comment:
Why are you using Math.abs here? Math.abs is to get the absolute value (https://www.w3schools.com/jsref/jsref_abs.asp), not necessary here. The function should be able to take negative numbers also